### PR TITLE
Update to nextcloud-vue 8 (beta)

### PR DESCRIPTION
**Pull request by susnux**
## Summary

Now that we migrated from every deprecated component for nextcloud-vue 8 we can update master to version 8 beta.

This includes all changes listed here https://github.com/nextcloud/nextcloud-vue/issues/4223

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- Screenshots before/after for front-end changes
- Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/settings/src/components/GroupListItem.vue [hunk 1]
```diff
@@ -90,9 +90,9 @@ export default {
 			required: true,
 		},
 		/**
-		 * Title of this group
+		 * Name of this group
 		 */
-		title: {
```
Comment (ShGKme) [inline apps/settings/src/components/GroupListItem.vue:95]: Was it intended or suddenly made with find and replace? :D

Comment (susnux) [reply apps/settings/src/components/GroupListItem.vue:95]: Both :sweat_smile: 
First a find and replace issue, but I think it makes sense to rename this property as it is basically the name and not the title.

**Comment by susnux** [comment]
BTW I have no idea why cypress with browserify broke, but changing to webpack as preprocessor works and reduces the npm dependencies (~80 dependencies less).

**Review by Pytal — APPROVED** [review]

**Review by ShGKme — APPROVED** [review]
Tested on:
- [x] Files
- [x] Sidebar, including
  - [x] Comments
  - [x] Versions
- [x] Settings, including
  - [x] Apps
  - [x] Users
  - [x] Theming
- [x] Weather status

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
Please rebase :see_no_evil:

**Pull request merged**